packages/constellation/src/components/Compositions/Table/Cells/InformationalCell/InformationalCell.types.ts

```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react'
+
+import { BaseCellProps } from '../..'
+
+interface Props extends BaseCellProps {
+  /**
+   * The text content displayed in the Tag
+   */
+  text: string
+  /**
+   * The stylistic variant to use when rendering the Tag.
+   */
+  variant?: 'default' | 'alt' | 'success' | 'warning' | 'error'
```

Comment:
Same props concern as above.

Reply:
Great point, will update these to import their props from the corresponding components now!